app/controllers/controllers.js

```diff
@@ -201,58 +201,74 @@ exports.showAllTeams = (req, res) => {
 
 // Handles teammate evaluation
 exports.evaluateTeammate = async (req, res) => {
-  if (req.session.user) {
-    var teammateID = req.params.id;
-    var reviewerID = req.session.user.id;
```

Comment:
You should add checks to ensure these variables are not null or undefined before accessing their properties.